src/components/PeopleFilters.tsx

```diff
@@ -27,66 +64,51 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {centuries.map(century => (
+              <SearchLink
+                key={century}
+                data-cy="century"
+                className={classNames('button mr-1', {
+                  'is-info': centuryFilter.includes(century),
+                })}
+                params={{
+                  centuries: centuryFilter.includes(century)
```

Comment:
The use of ternary operators inside the parameters of `SearchLink` makes the code a bit hard to read. It would be more readable if you extract this logic into a separate function or variable.

Reply:
Not fixed 
![image](https://github.com/mate-academy/react_people-table-advanced/assets/99403198/771ddb35-4cd5-416a-bd1d-87f34016edcf)